client/my-sites/email/email-providers-comparison/email-provider-details/style.scss

```diff
@@ -33,6 +33,25 @@
 	flex-wrap: wrap;
 }
 
+.email-provider-details__additional-price-information {
+	color: var( --color-text-subtle );
+	display: inline-block;
+	font-size: 12px;
```

Comment:
Nit: this should be `$font-body-extra-small`

Reply:
Good catch!